eth/api.go

```diff
@@ -1360,7 +1422,7 @@ func (s *PublicTransactionPoolAPI) SignTransaction(args SignTransactionArgs) (*S
 // the accounts this node manages.
 func (s *PublicTransactionPoolAPI) PendingTransactions() []*RPCTransaction {
 	pending := s.txPool.GetTransactions()
-	transactions := make([]*RPCTransaction, 0)
+	var transactions []*RPCTransaction
```

Comment:
This was added a while ago by @obscuren if I'm not mistaken together with the `resend` method.  The idea was a method that would return your submitted transactions that have not been processed due to a too low gas price. You can resubmit them with a higher price.